_component_design/buttons-drop-down.md

```diff
@@ -78,7 +78,7 @@ Use the following specifications when you construct a drop-down button.
 A drop-down button contains the following elements:
 
 - **Button (required)**: Refer to [buttons]({{site.baseurl}}/components/buttons.html) for more information about constructing a button.
-- **Angle-down icon (required)**: The drop-down menu appears when the user clicks the angle-down icon. See the [Icons](https://rackerlabs.github.io/helix-ui/components/icons/) page of the Helix Toolkit for information about the angle-down icon.
+- **Angle-down icon (required)**: The drop-down menu appears when the user clicks the angle-down icon. See the [Icons](https://rackerlabs.github.io/helix-ui/components/icon/) page of the Helix Toolkit for information about the angle-down icon.
```

Comment:
Modify link text as well. `Icons` -> `Icon`

Reply:
updated to singular